exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,8 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
-
+function containsCode(sentence) {
+  if(sentence = sentence.includes("code")){
+    return true;
+  }else {
+    return false;
+}
+}
```

Comment:
This is mostly correct, however you don't need to add `sentence =`. You can just do:
```suggestion
function containsCode(sentence) {
  if(sentence.includes("code")){
    return true;
  }else {
    return false;
}
}
```

Reply:
Got you. Thanks...